packages/itwinui-react/src/core/Tabs/Tabs.tsx

```diff
@@ -35,12 +40,13 @@ type TabsOverflowProps =
       /**
        * Type of the tabs.
        *
-       * If `orientation = 'vertical'`, `pill` is not applicable.
+       * If `type = 'pill' | 'borderless'`, `overflowOptions` is not applicable.
        * @default 'default'
```

Comment:
nit: Looks like `@default` appears twice when using the prop because VSCode combines both the JSDocs. Maybe we can remove one `@default` if it's not needed.

![image](https://github.com/iTwin/iTwinUI/assets/45748283/014eab60-9f3d-4ccc-95f1-07e4a8543e5d)

Reply:
Should be better